### PR TITLE
Testing documentation update.

### PR DESCRIPTION
**Pull request by dabail10**
Developer(s): D. Bailey
Updated documentation (Y/N): Y
Results (bit for bit, roundoff, climate changing): bfb
Code review: E. Hunke, A. Roberts
Other Relevant Details:
The Makefile is still working with my software stack. There are still some tweaks in the txt files
that I will work on, but overall the RST is close here.

### CODE REVIEW

**Comment by eclare108213** [comment]
Have these changes already been merged in previous PRs?  I don't know why there would be changes from 16 days ago still here.  Dave, I'm wondering if you're not updating your fork after previous PRs and/or creating new PRs when you could just add to older ones that haven't been satisfied yet.  ??

**Comment by dabail10** [comment]
Well, I did a pull from CICE-Consortium master. I'm a bit (git?) confused about why the pull requests seem to pull in more than I expected. It's like they are cumulative when previous PRs are not satisfied maybe?

**Comment by eclare108213** [comment]
I don't think it hurts to merge previous stuff again, it just makes the repo history hard to follow.  I'm not sure if any of this in PR#43 is redundant, actually -- what's confusing me is that the commit labels are very similar between this branch (master) and the PR#42 branch (gh-pages).  I request that commit labels be a bit more specific in the future, thanks.

**Comment by eclare108213** [comment]
I think that when you update your fork, git will pull in things that you previously committed, but they don't show up as potential changes to commit.  I could be wrong about that.
